.ci/scripts/test_llama.sh

```diff
@@ -130,9 +130,12 @@ cleanup_files() {
 prepare_artifacts_upload() {
   if [ -n "$UPLOAD_DIR" ]; then
     echo "Preparing for uploading generated artifacs"
+    # TODO(T197321521): Before there is a model/app agnostic test-spec.yml, we have to rename the
+    # model to be xnnpack_llama2.pte to make it recognizable by the existing test-spec
+    cp "${EXPORTED_MODEL_NAME}" xnnpack_llama2.pte
```

Comment:
Give me a min, let me try to update the spec file so that you don't need to hard code the model name here

Reply:
Is the apk and test suite hard-coded as well? @kirklandsign is merging the new activity, if we replace the app build step with it, would the test-spec work with it? By looking the test-spec code, I don't see it's hard-coded the app, but would like to check with you

Reply:
BTW can i see the test spec?

Reply:
I'm testing to see if this works https://ossci-assets.s3.amazonaws.com/android-llama2-device-farm-test-spec-v2.yml

Reply:
Testing the new spec https://github.com/pytorch/executorch/actions/runs/10327705480

Reply:
It turns out that the hack I have is a bit more involved to fix, so I create a separate PR here https://github.com/pytorch/executorch/pull/4657.  Pls take a look.  The current setup is just a dummy way to get the TPS value until we have the generic loader to properly run benchmark on devices.

Reply:
@guangy10 Once https://github.com/pytorch/executorch/pull/4657 lands, plz rebase this on top of that.  I have moved all the fixes to avoid hardcoding the model name there